### PR TITLE
Added minio port in sandbox

### PR DESCRIPTION
**Pull request by yindia**
# TL;DR
- Added Minio port in the sandbox. After https://github.com/flyteorg/flyte/pull/1681 we added a new port for the Minio console

We can merge it after flyte v0.18.0 patch release

Testing
- For now, I upgrade the helm chart from the master branch 

Screenshot 2021-10-19 at 9 10 19 PM

## Type
- [x] Bug Fix
- [ ] Feature
- [ ] Plugin

## Are all requirements met?

- [x] Code completed
- [x] Smoke tested
- [x] Unit tests added
- [x] Code documentation added
- [ ] Any pending items have an associated Issue

## Complete description
_How did you fix the bug, make the feature etc. Link to any design docs etc_

## Tracking Issue
https://github.com/flyteorg/flyte/issues/<number>

## Follow-up issue
_NA_
OR
_https://github.com/flyteorg/flyte/issues/<number>_

### CODE REVIEW

**Comment by evalsocket** [comment]
@pmahindrakar-oss new changes will be available after flyte patch release/ new release
